content/en/logs/workspaces/sql_reference.md

```diff
@@ -46,30 +46,57 @@ FROM orders {{< /code-block >}} |
+| `avg(numeric n)`                                 | numeric                               | Returns the average value (arithmetic mean) across all input values.        |
+| `ceil(numeric n)`                                | numeric                               | Returns the value rounded up to the nearest integer.                        |
+| `floor(numeric n)`                               | numeric                               | Returns the value rounded down to the nearest integer.                      |
+| `round(numeric n)`                               | numeric                               | Returns the value rounded to the nearest integer.                           |
+| `lower(string s)`                                | string                                | Returns the string as lowercase.                                            |
+| `upper(string s)`                                | string                                | Returns the string as uppercase.                                            |
+| `abs(numeric n)`                                 | numeric                               | Returns the absolute value.                                                 |
+| `coalesce(args a)`                               | typeof first non-null a OR null       | Returns the first non-null value or null if all are null.                   |
+| `cast(value AS type)`                            | type                                  | Converts the given value to the specified data type.                        |
+| `length(string s)`                               | integer                               | Returns the number of characters in the string.                             |
+| `trim(string s)`                                 | string                                | Removes leading and trailing whitespace from the string.                    |
+| `replace(string s, from_string s1, to_string s2)`| string                                | Replaces occurrences of a substring within a string with another substring. |
+| `substring(string s, start_position_int i, length_int l)` | string                        | Extracts a substring from a string, starting at a given position and for a specified length. |
+| `extract(field from timestamp/interval)`         | numeric                               | Extracts a part of a date or time field (e.g., year, month) from a timestamp or interval. |
+| `to_timestamp(numeric n)`                        | timestamp with time zone              | Converts a numeric value to a timestamp with time zone.                     |
+| `to_char(timestamp t / interval i / numeric n, format f)` | string                      | Converts a timestamp, interval, or numeric value to a string using a format.|
+| `date_trunct(field f, source [, time_zone])`     | timestamp [with time zone] / interval | Truncates a timestamp or interval to a specified precision.                 |
```

Comment:
```suggestion
| `date_trunc(field f, source [, time_zone])`     | timestamp [with time zone] / interval | Truncates a timestamp or interval to a specified precision.                 |
```

Reply:
@iFallUpHill Is this a typo?

Reply:
Yes, sorry the correction is accurate `DATE_TRUNC`